### PR TITLE
Remove powsybl-hades2-integration

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

File: src/main/resources/config/application.yaml [hunk 1]
```diff
@@ -40,6 +40,6 @@ powsybl-ws:
     customQuery: ${powsybl-ws.database.customQueryBegin}reWriteBatchedInserts=true
     name: sa
 
-# default values for loadflow providers "Hades2" or "OpenLoadFlow"
+# default values for security-analysis providers "Dynaflow" (SURE ?) or "OpenLoadFlow"
```
Comment (etiennehomer) [inline src/main/resources/config/application.yaml:43]: Only OLF is enough. Security analysis are not available with DynaFlow I think. To be checked

File: src/test/java/org/gridsuite/securityanalysis/server/SecurityAnalysisControllerTest.java [hunk 2]
```diff
@@ -597,7 +597,7 @@ public void getProvidersTest() throws Exception {
 
         resultAsString = mvcResult.getResponse().getContentAsString();
         List<String> providers = mapper.readValue(resultAsString, new TypeReference<List<String>>() { });
-        assertEquals(List.of("DynaFlow", "OpenLoadFlow", "Hades2"), providers);
+        assertEquals(List.of("DynaFlow", "OpenLoadFlow"), providers); // WHY Dynaflow ???
```
Comment (etiennehomer) [inline src/test/java/org/gridsuite/securityanalysis/server/SecurityAnalysisControllerTest.java:600]: Yes let's clean this

File: src/main/resources/config/application.yaml (hunk 1 above)
Comment (etiennehomer) [reply src/main/resources/config/application.yaml:43]: Actually it's available. So to be kept

Comment (etiennehomer) [reply src/main/resources/config/application.yaml:43]: So we should allow to use it in study-server ?

**Review by etiennehomer — APPROVED** [review]

**Review by AbdelHedhili — APPROVED** [review]

**Pull request merged**